experiments/AtmosGCM/heldsuarez.jl

```diff
@@ -117,6 +108,8 @@ function held_suarez_forcing!(bl, source, state, diffusive, aux, t::Real)
     _grav = FT(grav(bl.param_set))
     _cp_d = FT(cp_d(bl.param_set))
     _cv_d = FT(cv_d(bl.param_set))
+    #_p0 = FT(MSLP(bl.param_set))
+    _p0 = FT(1.01325e5)
```

Comment:
If this is needed, then please use
```
CLIMAParameters.Planet.MSLP(::EarthParameterSet) = 1.01325e5
```
above in global scope, somewhere after
```
struct EarthParameterSet <: AbstractEarthParameterSet end
```
and leave this as
```
_MSLP = FT(MSLP(bl.param_set))
```